src/main.rs

```diff
@@ -55,7 +61,13 @@ fn run() -> Result<(), MainError> {
             let manifest = read_package_manifest(&manifest_path)?;
             if let Some(name) = args.script {
                 if let Some(command) = manifest.scripts.get(&name) {
-                    eprintln!("> {command}");
+                    eprintln!(
+                        "\n> {}@{} {}",
+                        &manifest.name,
+                        &manifest.version,
+                        &cwd.display()
+                    );
+                    eprintln!("> {command}\n");
```

Comment:
I want you to test this part.

Reply:
You may not need to add a new test to cover this if there are existing test that already calls `pnpm run`.

Reply:
Apologies, i got it wrong. yeah i removed the wrong ones and added new tests in the `tests/test_main.rs` file.
the `run_script` file checks for the output to contain both package name and version while the `run_from_workspace_root` checks it without any name and version. so that should cover both the scenarios.